.goreleaser.yml

```diff
@@ -17,14 +17,14 @@ builds:
   ldflags:
   - -w -X github.com/tektoncd/cli/pkg/cmd/version.clientVersion={{.Version}}
 archives:
-- name_template: "{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
-  replacements:
-    darwin: Darwin
-    linux: Linux
-    windows: Windows
-    386: i386
-    amd64: x86_64
-    arm64: aarch64
+- name_template: >-
+    {{- .Binary }}_
+    {{- .Version }}_
+    {{- title .Os }}_
+    {{- if eq .Arch "amd64" }}x86_64
```

Comment:
are all the cases handled ?

Reply:
i.e.
```
{{- else if eq .Arch "darwin" }}MacOS
{{- else if eq .Arch "linux" }}Linux
{{- else if eq .Arch "windows" }}Windows
```

Reply:
I believe these are not possible values for Arch, right?